src/wallet/mod.rs

```diff
@@ -1548,19 +1551,60 @@ where
     }
 }
 
+/// Return a fake wallet that appears to be funded for testing.
+pub fn get_funded_wallet(
```

Comment:
Did you try with `#[cfg(test)]` to see if you could hide this to non-testing environments?

I didn't realize this yesterday, I thought this was still part of the `test` module, which is only present when testing is enabled.

Reply:
If I add this, then I get the following error in bdk-reserves: "unresolved import `bdk::wallet::get_funded_wallet`"

And if I try the following in bdk-reserves/Cargo.toml: "bdk = { ...  features = ["electrum", "test"] }" then I get the following error: "the package `bdk-reserves` depends on `bdk`, with features: `test` but `bdk` does not have these features."

It would be cool to make it work like this, but I don't know how.